tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py26, py27, py33, py34, py35, pypy
+
+[testenv]
+passenv = HYPOTHESIS_STORAGE_DIRECTORY
+
+deps =
+  nose-cov
+  hypothesis
+
+commands =
+  nosetests -v --logging-level=DEBUG --with-cov --cov chardet --cov-config .coveragerc
```

Comment:
I know and in fact it isn't. See below. The `commands` option is overridden for PyPy.